pkg/chunk/storage/factory_test.go

```diff
@@ -154,6 +157,45 @@ func TestCustomIndexClient(t *testing.T) {
 	}
 }
 
+func TestCassandraInMultipleSchemas(t *testing.T) {
```

Comment:
What are you testing specifically here? That the registered doesn't panic?

Reply:
yes, do you see any problem with it or the way it is tested?